### ISSUE TITLE
[Improvement]Tapping "+" in tabs tray does not always open a new tab

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#9369]
## Steps to reproduce
1. Open Tabs Tray
1. Tap on the "+" button
    1. Expected behavior

* A new tab should be open.
    1. Actual behavior
* It seems that sometimes tapping on the "+" icon a new tab is not open. My feeling is that we should increase the hot spots button.
    1. Device & build information
* Device: iPhone 12 Pro (15.0.2)
* Build version: 39.0 (6442)
    1. Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-3330)

**Comment by alexgibson** [comment]
I came here looking to file a very similar bug to this. Firefox on my iPhone has suffered from this issue for quite a long time. My assumption was this could be a performance issue, as the button seems often unresponsive when I first open the app. But I had not considered that it could just be a small touch target.

This seems like a minor issue on the surface, but for me it’s been the most frustration I’ve had with Firefox on iOS. It’s a small paper cut, but enough to have made me consider switching browser seeing as it’s a button I use all the time.

**Comment by data-sync-user** [comment]
➤ Asa Dotzler commented:

Touch target size should be 44 px square to ensure accessibility. This is an accessibility severity 3.
